src/shared/tests.h

```diff
@@ -3,6 +3,19 @@
 
 #include <stdbool.h>
 
+#include "macro.h"
+
+static inline bool manager_skip_test(int r) {
```

Comment:
maybe rename this to "manager_errno_skip_test()" or "manager_skip_test_errno()" while at it? if you rename this might as well clarify what this is for...

(or maybe even make this more like the macros from errno-util.h, i.e. name it ERRNO_IS_MANAGER_SKIP_TEST()?)

Reply:
manager_errno_skip_tests sounds nice. I prefer to reserve uppercase for stuff that is actually macros. Just because this is a simple inline function, it is still a function, and the caller doesn't care.